capstone/capapi/serializers.py

```diff
@@ -359,106 +419,164 @@ def to_representation(self, instance, check_permissions=True):
+        read_only=True,
+        lookup_field="slug",
+    )
+    jurisdiction = serializers.ReadOnlyField(source="jurisdiction.name")
     url = serializers.HyperlinkedIdentityField(
-        view_name="court-detail",
-        lookup_field='slug')
+        view_name="court-detail", lookup_field="slug"
+    )
 
     class Meta:
         model = models.Court
         fields = (
-            'id',
-            'url',
-            'name',
-            'name_abbreviation',
-            'jurisdiction',
-            'jurisdiction_url',
-            'slug',
+            "id",
+            "url",
+            "name",
+            "name_abbreviation",
+            "jurisdiction",
+            "jurisdiction_url",
+            "slug",
         )
 
 
 ### BULK SERIALIZERS ###
 
+
 class CaseExportSerializer(serializers.ModelSerializer):
     download_url = serializers.SerializerMethodField()
 
     class Meta:
         model = models.CaseExport
-        fields = ('id', 'download_url', 'file_name', 'export_date', 'public', 'filter_type', 'filter_id', 'body_format')
+        fields = (
+            "id",
+            "download_url",
+            "file_name",
+            "export_date",
+            "public",
+            "filter_type",
+            "filter_id",
+            "body_format",
+        )
 
     def get_download_url(self, obj):
-        return api_reverse('caseexport-download', kwargs={'pk': obj.pk}, request=self.context.get('request'))
+        return api_reverse(
+            "caseexport-download",
+            kwargs={"pk": obj.pk},
+            request=self.context.get("request"),
+        )
 
 
 class NoLoginCaseDocumentSerializer(CaseDocumentSerializerWithCasebody):
     def to_representation(self, instance, check_permissions=False):
-        """ Tell get_casebody not to check for case download permissions. """
+        """Tell get_casebody not to check for case download permissions."""
         return super().to_representation(instance, check_permissions=check_permissions)
 
     @property
     def data(self):
-        """ Skip tracking of download counts. """
+        """Skip tracking of download counts."""
         return super(DocumentSerializer, self).data
+
+
+class ConvertCaseDocumentSerializer(CaseDocumentSerializer):
+    first_page_order = serializers.CharField()
+    last_page_order = serializers.CharField()
+
+    def to_representation(self, instance):
+        first_page_order = self.context.get("first_page_order")
+        last_page_order = self.context.get("last_page_order")
+
+        data = super().to_representation(instance)
+
+        data.pop("reporter", None)
+        data.pop("volume", None)
+        data.pop("url", None)
+        data.pop("frontend_url", None)
+        data.pop("frontend_pdf_url", None)
+        try:
+            data["court"].pop("slug", None)
+            data["court"].pop("url", None)
+        except KeyError:
+            print("Cannot pop fields because 'court' doesn't exist")
+            pass
```

Comment:
I don't think you need `pass` here (you'd need it only if it were the only instruction in this block).